Setup/Product.wxs

```diff
@@ -547,6 +853,109 @@
     </DirectoryRef>
     <ComponentGroup Id="Component.ConEmuFiles">
           <ComponentRef Id="File.CER00" /><ComponentRef Id="File.CER01" /><ComponentRef Id="File.CER03" /><ComponentRef Id="File.CER04" /><ComponentRef Id="File.CER05" /><ComponentRef Id="File.CER06" /><ComponentRef Id="File.CER07" /><ComponentRef Id="File.CER08" /><ComponentRef Id="File.CER09" /><ComponentRef Id="File.CER10" />
+        <ComponentGroup Id="Component.NetStandard">
+            <ComponentRef Id="Microsoft.Win32.Primitives.dll" />
+            <ComponentRef Id="netstandard.dll" />
```

Comment:
💡 Maybe use a separate component group for **netstandard.dll** (.NET Standard 2.0), and leave the rest here (.NET Standard 1.x).

Reply:
I had no idea it was even different versions.  It was a file in release folder.  That was reason enough for me to include it. It looked like it was related to .net standard soooo....

Reply:
The group is for net standard.  I don't feel the need to create separate version groups.